packages/lesswrong/components/posts/PostsItemTitle.jsx

```diff
@@ -10,27 +9,19 @@ import { withRouter } from '../../lib/reactRouterWrapper.js';
 
 const styles = theme => ({
   root: {
-    color: "rgba(0,0,0,.95)",
-    display: "flex",
+    color: "rgba(0,0,0,.87)",
     position: "relative",
+    lineHeight: "2rem",
     zIndex: theme.zIndexes.postItemTitle,
     [theme.breakpoints.down('xs')]: {
       paddingLeft: 2,
     },
-    [theme.breakpoints.down('sm')]: {
-      justifyContent: "space-between",
-      flexGrow: 1,
-    }
-  },
-  title: {
     overflow: "hidden",
     textOverflow: "ellipsis",
     whiteSpace: "nowrap",
-    display: "inline-block",
     alignItems: "center",
-    fontSize: "1.3rem",
+    fontSize: "1.4rem",
```

Comment:
There were clashing redundant fontSize styles, the other one coming from `<Typography>`. To compensate for taking out the `<Typography>` element, I updated this with the styles it would have applied. But it looks like I might not have gotten the precedence order right.